megamek/src/megamek/common/EquipmentType.java

```diff
@@ -502,7 +502,7 @@ public boolean isSpreadable() {
         return spreadable;
     }
 
-    public int getToHitModifier() {
+    public int getToHitModifier(@Nullable Mounted<?> mounted) {
```

Comment:
There are some cases (particularly in AlphaStrike and BattleForce conversion code) where the Mounted is not available. In those cases a `null` is passed, which skips the check for the laser pulse module.